### DISCUSSION TITLE
How to remove multiple options at once

### DISCUSSION THREAD

**Discussion by AleJavierHY** [spatie/laravel-options#8] [Q&A] [answered]
Hello, excellent package. I am trying to create a list of dynamic options from an enum class.
I'm using reject() as the documentation describes, but I can only remove one option.
Is there a way to remove multiple options at once?g

**Comment by rubenvanassche** [comment, marked as answer]
Since reject takes a closure you can just do this:

```php
Options::forEnum(Hobbit::class)->reject(fn(Hobbit $hobbit) => in_array($hobbit, [Hobbit::Frodo, Hobbit::Sam]);
```
